### PR TITLE
NF: uses edit with block

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
To shorten slightly the code

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt [hunk 1]
```diff
@@ -793,7 +797,7 @@ open class CardBrowser :
                 }
             }
             val hasUnmarked = originalUnmarked.isNotEmpty()
-            CardUtils.markAll(java.util.ArrayList(notes), hasUnmarked)
+            CardUtils.markAll(ArrayList(notes), hasUnmarked)
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt:800]: unrelated

File: AnkiDroid/src/test/java/com/ichi2/anki/OnboardingFlagTest.kt [hunk 2]
```diff
@@ -36,9 +37,8 @@ class OnboardingFlagTest : RobolectricTest() {
     }
 
     @Before
-    fun setShowOnboardingPreference() {
-        AnkiDroidApp.getSharedPrefs(targetContext).edit().putBoolean(SHOW_ONBOARDING, true).apply()
-    }
+    fun setShowOnboardingPreference() =
+        AnkiDroidApp.getSharedPrefs(targetContext).edit { putBoolean(SHOW_ONBOARDING, true) }
```
Comment (BrayanDSO) [inline AnkiDroid/src/test/java/com/ichi2/anki/OnboardingFlagTest.kt:41]: functional change. This method didn't return anything before, nor it should

File: AnkiDroid/src/test/java/com/ichi2/libanki/sync/RemoteMediaServerTest.kt [hunk 3]
```diff
@@ -94,17 +95,16 @@ class RemoteMediaServerTest {
...
     @KotlinCleanup("use edit{} extension function")
-    private fun setCustomMediaServer(s: String) {
-        val userPreferences = AnkiDroidApp.getSharedPrefs(AnkiDroidApp.instance)
-        val e = userPreferences.edit()
-        e.putBoolean("useCustomSyncServer", true)
-        e.putString("syncMediaUrl", s)
-        e.apply()
-    }
+    private fun setCustomMediaServer(s: String) =
+        AnkiDroidApp.getSharedPrefs(AnkiDroidApp.instance)
```
Comment (BrayanDSO) [inline AnkiDroid/src/test/java/com/ichi2/libanki/sync/RemoteMediaServerTest.kt:103]: ditto + the KotlinCleanups above can be removed

**Review by BrayanDSO — CHANGES_REQUESTED** [review]

File: AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt (hunk 1 above)
Comment (ShridharGoel) [reply AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt:800]: I think we should allow unrelated changes if they tend to improve the code.

Comment (ShridharGoel) [reply AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt:800]: Correct, I meant few unrelated changes which don't clutter the actual changes should be fine.

Comment (ShridharGoel) [reply AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt:800]: Regarding the code history, maybe we can have code changes in a different commit if it is a part of the same PR.

File: AnkiDroid/src/test/java/com/ichi2/libanki/sync/RemoteMediaServerTest.kt [hunk 4]
```diff
@@ -94,16 +95,15 @@ class RemoteMediaServerTest {
     @KotlinCleanup("use edit{} extension function")
```
Comment (BrayanDSO) [inline AnkiDroid/src/test/java/com/ichi2/libanki/sync/RemoteMediaServerTest.kt:95]: can be removed

**Review by BrayanDSO — APPROVED** [review]
LGTM. only a cleanup to remove, but not worth blocking this, as it may get conflicts easily

**Review by mikehardy — APPROVED** [review]
LGTM ("Let's Get This Merged" :laughing: )

**Pull request merged**